deployment/terraform/iam.tf

```
@@ -15,10 +18,30 @@ data "aws_iam_policy_document" "ecs_task_execution_assume_role" {
 
 resource "aws_iam_role" "ecs_task_execution_role" {
   name               = "ecs${var.environment}TaskExecutionRole"
-  assume_role_policy = "${data.aws_iam_policy_document.ecs_task_execution_assume_role.json}"
+  assume_role_policy = "${data.aws_iam_policy_document.ecs_assume_role.json}"
 }
 
 resource "aws_iam_role_policy_attachment" "ecs_task_execution_role_policy" {
   role       = "${aws_iam_role.ecs_task_execution_role.name}"
   policy_arn = "arn:aws:iam::aws:policy/service-role/AmazonECSTaskExecutionRolePolicy"
 }
+
+data "aws_iam_policy_document" "ses_send_email" {
+  statement {
+    effect = "Allow"
+
+    resources = ["*"]
+    actions   = ["ses:SendRawEmail"]
+  }
+}
+
+resource "aws_iam_role" "ecs_task_role" {
```

This is different than an ECS Task Execution Role: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_IAM_role.html

---

I would consider renaming this resource so that it is application specific. Similar to the security group associated with an ECS service, if we had a background worker service, it is possible that the role associated with that task would be different than this one.

```suggestion
resource "aws_iam_role" "app_task_role" {
```